scripts/build-layout-scales.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const stripIndent = require('strip-indent');
+const mediaQueries = require('../src/media-queries');
+const allScales = require('../src/scales');
+
+function buildLayoutScales(config) {
+  const layoutScales = typeof config === 'object' ? config : Object.assign(allScales, config);
```

Comment:
Ah this is a little half hearted. if a custom `config` exists, use that instead of the default `scales`. It's untested and I may have misinterpreted how you were doing this for color.